data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,174 @@
+			`	"A trauma surgeon for the Unfettered Hai? I wonder if she planned this years in advance?"`
+				goto questions
+			label accept
+			`	"If Anoo went anywhere, it would be Darkcloak. The largest Unfettered hospital is there. Many soldiers come back from raids injured on Darkcloak. A trauma surgeon would have a lot of work to do."`
+				accept
+			
+mission "Acorn Delights: Part 5"
+	landing
+	name "Acorns for Anoo"
+	description "Bring Eeeya to <destination> to deliver <cargo> to her missing granddaughter."
+	source Darkcloak
+	passengers 1
+	cargo "acorn cookies" 3
+	to offer
+		has "Acorn Delights: Part 4: done"
+	on visit
+		dialog "You forgot Eeeya or some of her mountains of cookies. Maybe they're in one of your escorts?"
+	on offer
+		"salary: Acorn Delights Dividends" = 500
+		payment 30000
+		log `Helped the famous Hai chef Eeeya find her granddaughter, a trauma surgeon named Anoo. Both joined the Unfettered hoping they can, "set them on a more productive path." Received shares of Acorn Delights stock which pay 500 credits a day in dividends.`
+		conversation
+			`You're ushered quickly from the landing platform to a nearby hospital. Unfettered Hai nearby are muttering in their language, but you keep hearing the Hai word for, "cookies" as they eye Eeeya. Along the way, you notice an odd contrast of squalor and advanced machinery. It seems the Unfettered value function and progress, rather than aesthetics and comfort like their southern brethren.`
+			`	As you rise through the main tower of the hospital, you're surprised at how advanced it is. Despite their willingness to die in battle, the Unfettered seem to value the lives of the survivors. Perhaps this is to heal them quickly to fight again?`
+			`	You reach a high floor, filled with personal offices. Eeeya takes you to a door at the end of a long hallway.`
+			choice
+				`	(Open the door.)`
+			`	 After the door opens, Eeeya and Anoo run to each other's arms. A few minutes later, they begin to speak. Eventually the conversation turns to Anoo's new occupation.`
+			`	"Why did you come here, Anoo? The Unfettered hate us."`
+			`	"They don't hate US, grandma. They hate what we've become: frivolous, uninventive, and self-important. Our race is fading into history through a slow death of mediocrity. Most Unfettered see warfare as our only salvation.`
+			`	"We think of them as mindless brutes, but they're not! They have scientists, doctors, musicians, and writers. Our resources dwarf theirs, but they thrive with what little they have. Soon the Unfettered will have many more planets and almost unlimited resources. The era of scarcity will be over, and they'll have the power to choose their future.`
+			`	"If enough good people join the Unfettered, we can change their minds; set them on a more productive path. They'll need good doctors."`
```

Comment:
```suggestion
			`	"If enough good people join the Unfettered, we can change their minds and set them on a more productive path. They'll need good doctors."`
```
You _could_ keep this as is because it's dialogue, but I don't see any stylistic reason to do so.
Talking big picture, I agree with Amazinite that the way this is worded doesn't really make sense considering what Anoo has said so far. Maybe you could clarify Anoo wants to `change their minds` in regards to warmongering, not everything.